src/tests/status.rs

```diff
@@ -64,3 +66,19 @@ $"#,
   assert!(re.is_match(&out), out);
   Ok(())
 }
+
+#[test_device]
+fn output_multiple(_model: nitrokey::Model) -> anyhow::Result<()> {
+  let devices = nitrokey::list_devices()?;
+  let res = Nitrocli::new().handle(&["status"])?;
```

Comment:
I guess I missed that before, but can we use `out` or some other name instead of `res`? The only association I have with `res` is result and in Rust code I typically try to use that only for `Result` objects. In any case, I believe something indicating that we are dealing with the program output makes more sense here.

Reply:
Sure.